lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -26,6 +26,10 @@ def self.available?
         File.exist?("#{BASE_DIR}/bin/kafka-run-class.sh")
       end
 
+      def self.configured?
+        MessageServerConfiguration.configured? || MessageClientConfiguration.configured?
+      end
+
```

Comment:
Would it be better to use `&&` here so that we return messaging as configured if both the client and server are configured?

Reply:
There should only be one appliance in a region configured as a server and the rest should be clients so `||` is fine.